kjnodes/chains.json

```diff
@@ -88,7 +88,7 @@
       "address": "bcnavaloper1aym6s8eza7kjvnxuwxufrzccz6vqvgnsc47cc7",
       "restake": {
         "address": "bcna1murx57yjv5r5gvspcxxd8d3w0mmy52xtdyreqd",
-        "run_time": "every 5 minutes",
+        "run_time": ["every 5 minutes"],

```

Comment:
REStake should support this but the array format is intended for multiple specific run times, not a single frequency like this.